qiskit_nature/problems/second_quantization/lattice/FermiHubbard.ipynb

```diff
@@ -0,0 +1,1270 @@
+      "text/plain": [
+       "<Figure size 432x288 with 1 Axes>"
+      ]
+     },
+     "metadata": {}
+    }
+   ],
+   "metadata": {
+    "slideshow": {
+     "slide_type": "subslide"
+    }
+   }
+  },
+  {
+   "cell_type": "markdown",
+   "source": [
+    "Labels are drawn for each node when with_labels=True."
```

Comment:
```suggestion
    "Labels are drawn for each node when setting `with_labels=True`."
```

Reply:
Similar changes need to be applied to more cells below. I will not comment on each occurrence manually from now on.